examples/mobile/UIComponents/components/controllers/index.html

```diff
@@ -87,6 +87,11 @@ <h1>
 						Chips
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="../list/reorder-list.html">
+						Reorder Button
```

Comment:
Is the Reorder Button same with Reorder List?

Reply:
Yes. It is the similar like case with expand button.